packages/askar/src/AskarModule.ts

```diff
@@ -0,0 +1,33 @@
+import type { DependencyManager, Module } from '@aries-framework/core'
+
+import { AriesFrameworkError, InjectionSymbols } from '@aries-framework/core'
+
+import { AskarStorageService } from './storage'
+import { AskarWallet } from './wallet'
+
+export class AskarModule implements Module {
+  public register(dependencyManager: DependencyManager) {
+    try {
+      // eslint-disable-next-line import/no-extraneous-dependencies
+      require('@hyperledger/aries-askar-nodejs')
+    } catch (error) {
+      try {
+        require('@hyperledger/aries-askar-react-native')
+      } catch (error) {
+        throw new Error('Could not load aries-askar bindings')
+      }
+    }
```

Comment:
👍 nice. We could probably improve this at some point by detecting the env and loading packages based on that. I think it could succeed in requireing the nodejs package even though we're in a RN env?

Reply:
I'm not convinced about this way of importing it but I wanted to left it consistent with the way we are using other shared components. And it's nice that it attempts to make life easier to developers using the framework.. unless they import nodejs package when on react native environment :open_mouth: . Would be nice to investigate if there is a reliable way to get the JS environment nowadays.

Reply:
Agreed. Also not 100% convinced yet. I think it's just another thing the user needs to provide to the agent config, while we could just infer it. Maybe we only try to import it if it's not provided yet? This way it 'works' out of the box. but you can also manually configure it